clean_project/search.py

```diff
@@ -0,0 +1,60 @@
+"""
+Search a word in a file and return all the occurrence specified the line numbers
+"""
+import argparse
+import psycopg2
+
+BOLD = '\033[1m'
+END = '\033[0m'
+
+
+def search(search_word):
+    """
+    Search if a given word is in bucket of files
+    :param search_word: Word given to search
+    :return: List of files and numbers of file when search_word occurrence
+    """
+    connection = None
+    result_search = []
+    try:
+        connection = psycopg2.connect(database='index_db', user='gianluca', password='password')
```

Comment:
I think that the connection can be used in a with statement. That would probably be the best way to use it.